tests/unauthorized/appsTest.js

```diff
@@ -38,11 +38,25 @@ describe('Unauthorized: Currently testing apps management,', function () {
       .willRespondWith(unauthorizedResponseObject)
   }
 
+  const appRequest = async (provider, method, app) => {
```

Comment:
Just to be a bit more specific
```suggestion
  const appRequestInvalidAuth = async (provider, method, app) => {
```